.travis.yml

```diff
@@ -17,6 +23,10 @@ jobs:
     - <<: *xenial-mixin
       python: 3.7
 
+    - stage: deploy
+      python: 3.6
+      script: "python -m statik.cmdline --upload=netlify --netlify-site-id=47df04ea-4f33-41a5-955e-3f48d6680b9d -p examples/blog"
```

Comment:
looking great.

`python -m statik --...` doesnt work?  `.cmdline` is needed?  that is a bug.

the hyperlinks in https://5bfb9b11792f895827ef832e--jolly-noyce-463b1a.netlify.com/ are not working.
can you double check that is a separate issue, and raise as a bug.

also use a travis environment variable instead of `47df04ea-4f33-41a5-955e-3f48d6680b9d` , as the main repo will use a different site, and other forks will need to create their own also, and set a different value for their travis builds.

Reply:
> the hyperlinks in https://5bfb9b11792f895827ef832e--jolly-noyce-463b1a.netlify.com/ are not working.
can you double check that is a separate issue, and raise as a bug.

Links seem to work for me. Some links do go to nowhere, but it seems like it was intentional, from the templates. Should I still raise the bug report?

> `python -m statik --...` doesnt work? .cmdline is needed? that is a bug.

`python -m statik` doesn't work locally for me. There isn't a `__main__` entry point.

> also use a travis environment variable instead of 47df04ea-4f33-41a5-955e-3f48d6680b9d , as the main repo will use a different site, and other forks will need to create their own also, and set a different value for their travis builds.

Done. https://travis-ci.org/kx-chen/statik/jobs/461611795

Reply:
> but it seems like it was intentional, from the templates. Should I still raise the bug report?

Yes

> python -m statik doesn't work locally for me. There isn't a __main__ entry point.

Another bug report.

Later today, I'll set up a netlify site and set NETLIFY_SITE_ID for merging.

Then we can look at your git+ requirement

Reply:
Issues made.